README.md

```diff
@@ -9,26 +9,65 @@ cannot find it out in the environment.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+1. Add this gem to your Gemfile
 
-    gem 'flowdock-build-notifier'
+2. Make a `.flowdock_build_notifier.yml` file in the root of your project. An
+   example is given below: 
 
-And then execute:
+```yaml
+flowdock_user_token: SECRET_FLOWDOCK_USER_TOKEN
+flow_name: FLOW_NAME
 
-    $ bundle
+teamcity_url: https://teamcity.your_account.com
+teamcity_user: SECRET_TEAMCITY_USER_NAME
+teamcity_password: SECRET_TEAMCITY_USER_PASSWORD
 
-Or install it yourself as:
+email_map:
+  miranda@gmail.com: miranda@company.com
+  jj@gmail.com: jacobson@company.com
+```
 
-    $ gem install flowdock_notify
+For privacy, you should put this file in your .gitignore.
+TeamCity login information and flowdock user token can be left
+out of the yaml configuration file by assigning the following ENV variables 
+on TeamCity if you prefer.
 
-## Usage
+```
+ENV['FLOWDOCK_USER_TOKEN']
+ENV['FLOWDOCK_NOTIFIER_TEAMCITY_USER']
+ENV['FLOWDOCK_NOTIFIER_TEAMCITY_PASSWORD']
+```
 
-TODO: Write usage instructions here
+The email map is used to connect the author of the last commit to the
+corresponding Flowdock account.
 
-## Contributing
+3. On TeamCity, set up your build configuration steps to run `bundle install`
+   initially. Make a final step to trigger regardless of previous step outcomes
+   and use the following as a meta-runner: 
```

Comment:
this is kinda "huh?" so it may be good to add screenshots or a link to what a meta-runner is or how to set it up